ui/src/app/stores/workflows/workflows.reducers.ts

```diff
@@ -496,6 +496,35 @@ export function workflowsReducer(state: State = initialState, action: WorkflowsA
         ...state,
         loading: false,
       };
+    case WorkflowsActions.REVERT_WORKFLOW:
+      return {
+        ...state,
+        workflowAction: {
+          ...initialState.workflowAction,
+          mode: workflowModes.REVERT,
+          id: action.payload,
+          loading: true,
+        },
+      };
+    case WorkflowsActions.REVERT_WORKFLOW_SUCCESS:
+      return {
+        ...state,
+        workflowAction: {
+          ...state.workflowAction,
+          loading: false,
+          workflow: undefined,
```

Comment:
Does this not have to be `action.payload`?

Reply:
Not really, You would not be able to update a workflow via form because we compare workflow and workflowForForm